airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeInternalStagingSqlOperations.java

```diff
@@ -30,24 +28,52 @@ public class SnowflakeInternalStagingSqlOperations extends SnowflakeSqlStagingOp
       "CREATE STAGE IF NOT EXISTS %s encryption = (type = 'SNOWFLAKE_SSE') copy_options = (on_error='skip_file');";
   private static final String PUT_FILE_QUERY = "PUT file://%s @%s/%s PARALLEL = %d;";
   private static final String LIST_STAGE_QUERY = "LIST @%s/%s/%s;";
-  private static final String COPY_QUERY = "COPY INTO %s.%s FROM '@%s/%s' "
-      + "file_format = (type = csv compression = auto field_delimiter = ',' skip_header = 0 FIELD_OPTIONALLY_ENCLOSED_BY = '\"' NULL_IF=('') )";
+  // the 1s1t copy query explicitly quotes the raw table+schema name.
+  private static final String COPY_QUERY_1S1T =
+      """
+      COPY INTO "%s"."%s" FROM '@%s/%s'
+      file_format = (
+        type = csv
+        compression = auto
+        field_delimiter = ','
+        skip_header = 0
+        FIELD_OPTIONALLY_ENCLOSED_BY = '"'
+        NULL_IF=('')
+      )""";
+  private static final String COPY_QUERY = """
+      COPY INTO %s.%s FROM '@%s/%s'
+      file_format = (
+        type = csv
+        compression = auto
+        field_delimiter = ','
+        skip_header = 0
+        FIELD_OPTIONALLY_ENCLOSED_BY = '"'
+        NULL_IF=('')
+      )""";
   private static final String DROP_STAGE_QUERY = "DROP STAGE IF EXISTS %s;";
   private static final String REMOVE_QUERY = "REMOVE @%s;";
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SnowflakeSqlOperations.class);
 
   private final NamingConventionTransformer nameTransformer;
+  private final boolean use1s1t;
 
   public SnowflakeInternalStagingSqlOperations(final NamingConventionTransformer nameTransformer) {
     this.nameTransformer = nameTransformer;
+    this.use1s1t = TypingAndDedupingFlag.isDestinationV2();
   }
 
   @Override
   public String getStageName(final String namespace, final String streamName) {
-    return nameTransformer.applyDefaultCase(String.join(".",
-        nameTransformer.convertStreamName(namespace),
-        nameTransformer.convertStreamName(streamName)));
+    if (use1s1t) {
```

Comment:
Is something that only applies to 1s1t?

Reply:
yep. this prevents snowflake from uppercasing the schema/table name, which is a breaking change relative to current behavior.